core/trino-main/src/main/java/io/trino/execution/scheduler/NodeScheduler.java

```diff
@@ -158,7 +158,7 @@ public static SplitPlacementResult selectDistributionNodes(
             NodeMap nodeMap,
             NodeTaskMap nodeTaskMap,
             long maxSplitsWeightPerNode,
-            long maxPendingSplitsWeightPerTask,
```

Comment:
that should be squased with commit that adds actual feature. We sometimes revert things, so if this change is separate, them we might miss it